components/esp_system/include/esp_private/usb_console.h

```diff
@@ -58,6 +58,8 @@ ssize_t esp_usb_console_available_for_read(void);
 
 bool esp_usb_console_write_available(void);
 
+bool esp_usb_console_read_available(void);
```

Comment:
This fixed some build errors during CDC. The only build error left is the Static Assert about dual core.